### PR TITLE
 Update K8s getting-started guide

### PR DESCRIPTION
**Pull request by sabbyanandan**
This is a continuation of what @cppwfs started in https://github.com/cppwfs/spring-cloud-dataflow/commit/2d64ce82a585416be79256f1030bd684844bbb2a, but this PR also addresses the concerns around the stream-update experience on K8s.

Resolves spring-cloud/spring-cloud-dataflow#2962

### CODE REVIEW

File: spring-cloud-dataflow-docs/src/main/asciidoc/getting-started-cloudfoundry.adoc [hunk 1]
```diff
@@ -265,7 +265,7 @@ spring.cloud.dataflow.task.platform.cloudfoundry.accounts[default].connection.sk
 # Note however that when the *server* is running locally, it can't access that db
 # task related commands that show executions won't work then
 spring.cloud.dataflow.task.platform.cloudfoundry.accounts[default].deployment.services=mysqlcups
-skipper.cliet.host=https://<skipper-host-name>/api
+skipper.cliet.serverUri=https://<skipper-host-name>/api
```
Comment (cppwfs) [inline spring-cloud-dataflow-docs/src/main/asciidoc/getting-started-cloudfoundry.adoc]: cliet should be client.

**Comment by cppwfs** [comment]
LGTM.

**Comment by chrisjs** [comment]
LGTM, merging

**Comment by chrisjs** [comment]
merged in 534d172

**Pull request closed without merging**